addons/skin.estuary/xml/Variables.xml

```diff
@@ -798,4 +798,7 @@
 		<value condition="ListItem.IsStereoscopic">$INFO[ListItem.Duration,$LOCALIZE[180]: ][CR]$VAR[VideoCodecVar,, ]$INFO[ListItem.VideoResolution,| , ]$VAR[VideoResolutionTypeVar,, ]$VAR[VideoHDRVar,| , ]| 3D $INFO[ListItem.VideoAspect,| ,:1 ]$VAR[AudioCodecVar,| , ]$VAR[AudioChannelsVar]</value>
 		<value>$INFO[ListItem.Duration,$LOCALIZE[180]: ][CR]$VAR[VideoCodecVar,, ]$INFO[ListItem.VideoResolution,| , ]$VAR[VideoResolutionTypeVar,, ]$VAR[VideoHDRVar,| , ]$INFO[ListItem.VideoAspect,| ,:1 ]$VAR[AudioCodecVar,| , ]$VAR[AudioChannelsVar]</value>
 	</variable>
+	<variable name="PVRInstanceName">
+		<value condition="System.PVRCount">$INFO[ListItem.backendinstancename,[COLOR grey]$LOCALIZE[31137]:[/COLOR] ,[CR]]</value>
```

Comment:
Just a nitpick: I would prefer camel case for the label name => BackendInstanceName

Reply:
Done.